src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepository.kt

```diff
@@ -59,6 +85,22 @@ class AktivitetskravRepository(private val database: DatabaseInterface) {
             ORDER BY created_at DESC
             """
 
+        private const val CREATE_AKTIVITETSKRAV_NY_VURDERING =
```

Comment:
Det er vel egentlig ikke noe spesielt med denne som gjør at den trenger hete NY_VURDERING. Ideelt sett burde vi samle funksjon og query som gjør INSERT INTO AKTIVITETSKRAV ett sted - lag gjerne ny PR eller lapp på det.

Reply:
https://trello.com/c/btvELVYu/1783-samle-funksjoner-for-lagring-av-aktivitetskrav-i-isaktivitetskrav